src/geometry.rs

```diff
@@ -1722,4 +1772,109 @@ mod tests {
 
         assert!(geometry.has_no_orphan_vertices());
     }
+
+    #[test]
+    fn test_triangle_face_to_reverted_should_pass_because_compared_to_reverted() {
+        let face = TriangleFace::new_separate(1, 2, 3, 4, 5, 6);
+        let face_reverted_correct = TriangleFace::new_separate(3, 2, 1, 6, 5, 4);
+
+        let face_reverted_calculated = face.to_reverted();
+        assert_eq!(face_reverted_correct, face_reverted_calculated);
+    }
```

Comment:
not shuffled! shifted. two faces with shifted lists of referenced vertices and normals are identical. the tests do prove this works, also because they are covering also inequality (see the following test).

Reply:
What I'm trying to say is that I'd expect you to test if `to_reverted` really reverted verices and normals. In this case, because of how partial equality works, all of these tests would pass:

```
assert_eq!(
    face_reverted_calculated,
    TriangleFace::new_separate(3, 2, 1, 6, 5, 4)
);
assert_eq!(
    face_reverted_calculated,
    TriangleFace::new_separate(1, 3, 2, 4, 6, 5)
);
assert_eq!(
    face_reverted_calculated,
    TriangleFace::new_separate(2, 1, 3, 5, 4, 6)
);
```

I know it's technically true, but at the same time, I'd expect this test to pass only the first case to showcase that reverted vertices and normals are **exactly equal**. For all we know, there might be some bug reverting these values incorrectly to e.g.  `(1, 3, 2, 4, 6, 5)` and this test case wouldn't catch it.

Reply:
I understand. We can test the reverted face to be the exact mirrored-image of the original, but what would it be good for? Semantically `(1, 3, 2, 4, 6, 5)` is a reverted version of `(1, 2, 3, 4, 5, 6)`.

Reply:
That's of course right, but since this is an unit test of function that should produce precise order of numbers, I'd lean towards exact equality.

Maybe @yanchith has something to say about this as well since he had an opinion regarding partial equality.

Reply:
well, if you insist, I'll check the result field by field. I still think this is an unreasonable privilege of a technicality over a true nature of the data.

Reply:
I don't insist, it's just a gut feeling. That's why I'd like to hear another opinion.